x/evm/keeper/state_transition.go

```diff
@@ -79,6 +79,9 @@ func (k *Keeper) NewEVM(
 	tracer vm.EVMLogger,
 	stateDB vm.StateDB,
 ) evm.EVM {
+	// Use BlockHash for random field
+	random := common.BytesToHash(ctx.HeaderHash())
```

Comment:
looks like the random number generated corresponds to the [RANDAO](https://eth2.incessant.ink/book/06__building-blocks/02__randomness.html#enter-randao) and not the block header. 

The header hash is not random by definition

Reply:
@austinchandra if we can't add RANDAO (most likely blocked on ABCI++ VoteExtension), I'd suggest adding a note saying that the RANDOM opcode is currently not supported.

Reply:
Do we have a doc page listing unsupported features vs. Ethereum?